tests/basic.spec.ts

```diff
@@ -512,6 +512,16 @@ test(scoped`automatic identify url text`, async ({ page }) => {
 <affine:page>
   <affine:note
     prop:background="--affine-background-secondary-color"
+    prop:edgeless={
```

Comment:
This significantly bloats almost all test cases, which is very much a code smell. @Saul-Mirone any idea avoiding the duplicate here?